igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -407,7 +408,7 @@
    "id": "50df4f46-e91a-4d04-a522-c832ff6fe4e7",
    "metadata": {},
    "source": [
-    "Now that we've covered the basics of our MintPy configuration, it's time to perform our analysis! As mentioned, above we will be processing data from four stacks, so you'll need to run the section of code starting at section 3.3 (here) to just before section 4 four times. Once for each data stack. Make sure to run each stack in order by modifying the cell below."
+    "Now that we've covered the basics of our MintPy configuration, it's time to perform our analysis! As mentioned above, we will be processing data from four stacks, so you'll need to run the section of code starting here at section 3.3 to just before section 4 four times, once for each data stack. Make sure to run each stack in order by modifying the stack variable in the cell below."
```

Comment:
@forrestfwilliams Is the stack variable the only thing that changes from one run to the next? If so, it would be helpful to list explicitly the four values they will need to use. I would have assumed it was [0, 1, 2, 3], but based on the change from 0 to 1 proposed in this PR, is it [1, 2, 3, 4]? Guidance could also be provided in the comment on line 421.